sechub-administration/src/test/java/com/mercedesbenz/sechub/domain/administration/project/ProjectServiceTest.java

```diff
@@ -82,12 +102,23 @@ void user2_sees_assigned_and_owned_projects_with_users() {
 
         assertThat(projects.get(0).getAssignedUsers().size()).isGreaterThan(0);
         assertThat(projects.get(1).getAssignedUsers().size()).isGreaterThan(0);
+
+        // verify profile IDs are included correctly
+        for (ProjectData projectData : projects) {
+            if (PROJECT_ID_2.equals(projectData.getProjectId())) {
```

Comment:
you should add a sanity check here - if the PROJECT_ID_2 would be never inside the results the assertion would never happen...

I would do it the following way:
```java
 boolean assertionDone = false;
 for (ProjectData projectData : projects) {
            if (PROJECT_ID_2.equals(projectData.getProjectId())) {
assertThat(projectData.getAssignedProfileIds()).containsAll(profileIdsProject2);
assertionDone=true;
          }
}
assertThat(assertionDone).isTrue();